.gitignore

```diff
@@ -0,0 +1,6 @@
+# Ignore secrets folder
+secrets/*
+experimental_docker_compose_files/02_custom_docker_file_connecting_agent_and_controller/secrets/*
+# Ignore dockerfiles because get's changed during the gitpodURL.sh
+dockerfiles/jenkins.yaml
```

Comment:
If you ignore `dockerfiles`, the changes you'll make in them won't be part of the repo.